src/parser/parser.yy

```diff
@@ -2720,11 +2720,22 @@ create_space_sentence
         auto sentence = new CreateSpaceSentence($4, $3);
         $$ = sentence;
     }
+    | KW_CREATE KW_SPACE opt_if_not_exists name_label KW_ON name_label {
+        auto sentence = new CreateSpaceSentence($4, $3);
+        sentence->setGroupName($6);
+        $$ = sentence;
+    }
```

Comment:
A little question， Why not use attribute value？ such as : 
CREATE SPACE default_space(partition_num=9, replica_factor=3, group_name="group0")

Reply:
Maybe it's a different design. In my opinion, group name is not a property for space

Reply:
> Maybe it's a different design. In my opinion, group name is not a property for space

Got, nice. LGTM.